components/footer/footer.stories.tsx

```diff
@@ -11,6 +11,6 @@ const Template: Story<FooterProps> = (args) => <Footer {...args} />;
 
 export const footer = Template.bind({});
 footer.args = {
-  homeBtn: "https://image.flaticon.com/icons/png/128/1946/1946436.png",
-  favsBtn: "https://image.flaticon.com/icons/png/128/535/535234.png",
+  homeIcon: "https://image.flaticon.com/icons/png/128/1946/1946436.png",
+  favsIcon: "https://image.flaticon.com/icons/png/128/535/535234.png",
```

Comment:
Unfortunately I don't have a preview, but is it necessary to save the images as PNG? If also JPG goes, I would prefer that because of the size

Reply:
It is actually an interesting topic. I tries to import personalised icons as .svg from the local public folder but for some reason it  doesn't seem to work. If you have any ideas, feel free to suggest :)

Reply:
Sorted!! ✅